crates/sncast/src/starknet_commands/verify/mod.rs

```diff
@@ -0,0 +1,138 @@
+use anyhow::{anyhow, bail, Result};
+use base::VerificationInterface;
+use camino::Utf8PathBuf;
+use clap::{Args, Parser, ValueEnum};
+use promptly::prompt;
+use scarb_api::StarknetContractArtifacts;
+use sncast::response::structs::VerifyResponse;
+use sncast::Network;
+use starknet::core::types::Felt;
+use std::collections::HashMap;
+use std::fmt;
+use voyager::VoyagerVerificationInterface;
+use walnut::WalnutVerificationInterface;
+
+pub mod base;
+mod voyager;
+mod walnut;
+
+#[derive(Args, Debug, Clone)]
+#[group(required = true, multiple = false)]
+pub struct ContractAddressOrClassHashGroup {
+    /// Contract address of the contract. Either this or class hash should be provided.
+    #[clap(short = 'd', long)]
+    pub contract_address: Option<Felt>,
+
+    /// Class hash of the contract. Either this or contract address should be provided.
+    #[clap(short = 'x', long)]
+    pub class_hash: Option<Felt>,
+}
+
+#[derive(Parser)]
+#[command(about = "Verify a contract through a block explorer")]
+pub struct Verify {
+    #[clap(flatten)]
+    pub contract_address_or_class_hash: ContractAddressOrClassHashGroup,
+
+    /// Class name of the contract to be verified. Either this or class hash should be provided.
```

Comment:
This is required, no?

Reply:
Yes, that's a mistake on my part, will amend it now.